sccp/figures/commonFuncs/plotUMAP.py

```diff
@@ -81,6 +81,7 @@ def points(
         assert values.shape[0] == points.shape[0]
 
         min_val, max_val = np.min(values), np.max(values)
+        #Here!
         bin_size = (max_val - min_val) / 255.0
         data["val_cat"] = pd.Categorical(
```

Comment:
@aarmey This is the issue that I was talking about. Since we are enforcing the bounds to between the min and max value of the weighted projections, the diverging palette can be confusing since the middle values won't always be 0. The plot this corresponds to is figureLupus3.py